### DISCUSSION TITLE
[@types/leaflet] v1.8.0+ not playing nice with @types/leaflet.locatecontrol

### DISCUSSION THREAD

**Discussion by TurtIeSocks** [DefinitelyTyped/DefinitelyTyped#63516] [Issues with a @types package]
I dug through the commits but can't seem to find why the latest `@types/leaflet` are now not playing nicely with the `@types/leaflet.locatecontrol` types. 

Working:
v1.79.11

Not working:
v1.8.0, v1.9.0

The module augmentation seems to stop working. If I copy the contents of `@types/leaflet.locatecontrol` to a local file outside of `node_modules`, then it works fine. I went back and forth between versions several times, with two different repos to verify that it's not a different package or TypeScript configuration issue. Everything works perfectly fine with v1.79.11 but consistently breaks when I try v1.8.0 or v1.9.0. 
<img width="501" alt="Screenshot 2022-12-07 at 12 14 38 AM" src="https://user-images.githubusercontent.com/58572875/206094363-416d25b0-2e97-4ea0-82de-9ba0c746afee.png">

[a comment by an automated account is not shown]

**Comment by someonewithpc** [comment]
One thing I can see in @types/leaflet.locatecontrol that seems a bit weird to me is the `declare module 'leaflet'`, but it seems to always have been that way. How are you testing this?

Also, `L.control` is a namespace, so it seems weird that it's saying it's a type

**Comment by cdauth** [comment]
I think this is obsolete, since the latest versions of leaflet.locatecontrol only export their classes as part of an ESM module. It doesn’t add itself to the `L` object anymore.

**Reply by someonewithpc** [reply]
Indeed https://github.com/domoritz/leaflet-locatecontrol/commit/49e108eee4a5ace111a330c4ae10396ea71bda6f#diff-1e64fa8dc305733bf5225c601d27b39d8e8e6b902405098ef47d59ba95eb179fL950
